packages/wrangler/src/cli.ts

```diff
@@ -1,5 +1,38 @@
+import * as Sentry from "@sentry/node";
+import { RewriteFrames } from "@sentry/integrations";
+import process from 'process';
 import { hideBin } from "yargs/helpers";
+
 import { main } from ".";
+import * as pkj from "../package.json"
+
+global.__rootdir__ = __dirname || process.cwd();
```

Comment:
This will leak user data. Is there a better option? 

Also, why are we attaching it to global? It appears to be directly referenced in this file, so can't it be a regular let/const?

Reply:
Seemed like it was necessary for TypeScript, I can try running some test runs without it. This is what I was basing it off https://docs.sentry.io/platforms/node/typescript/#changing-events-frames

Reply:
Seems like I can still see the Source just fine without it. I will drop that line.
<img width="494" alt="Screen Shot 2022-01-11 at 10 51 38 AM" src="https://user-images.githubusercontent.com/27247160/148986130-6eec70f4-4479-40ac-b643-7379933415e7.png">

Reply:
removed the `RewriteFrames` integration which was referencing the "global rootdir"